src/backend/columnar/.gitignore

```diff
@@ -1,68 +0,0 @@
-# =====
-# = C =
-# =====
-
-# Object files
-*.o
-*.ko
-*.obj
-*.elf
-*.bc
-
-# Libraries
-*.lib
-*.a
-
-# Shared objects (inc. Windows DLLs)
-*.dll
-*.so
-*.so.*
-*.dylib
-
-# Executables
-*.exe
-*.app
-*.i*86
-*.x86_64
-*.hex
-
-# ========
-# = Gcov =
-# ========
-
-# gcc coverage testing tool files
-
-*.gcno
-*.gcda
-*.gcov
-
-# ====================
-# = Project-Specific =
-# ====================
-
-/data/*.cstore
-/data/*.footer
-
-/sql/*block_filtering.sql
-/sql/*copyto.sql
-/sql/*create.sql
-/sql/*data_types.sql
-/sql/*load.sql
-
-/expected/*block_filtering.out
-/expected/*copyto.out
-/expected/*create.out
-/expected/*data_types.out
-/expected/*load.out
-/results/*
-/.deps/*
-/regression.diffs
-/regression.out
-.vscode
-
-*.pb-c.*
-
-# ignore files that could be created by circleci automation
-files.lst
```

Comment:
Is this ignored by any other gitignore?  

It seems like this file isn't covered entirely by other git files( with a quick look). Have you checked the rules and realized this is not necessary?

Reply:
I've seen that some of the lines (like you shared) were not covered in other gitignore files.
Then installed citus/executed regression tests etc. and verified that there were no such junk files, so I thought that they might be left over from old cstore repo or something

Reply:
They might be leftovers or they might be files that are generated after some runs, or in certain environments. It is a bit hard to be sure, I just wanted to point out this possibility.

Reply:
> It is a bit hard to be sure, I just wanted to point out this possibility.

yeah that makes sense, let's wait for @jeff-davis's input then